native/src/components/PrivacyCheckbox.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import styled from 'styled-components/native'
+
+import buildConfig from '../constants/buildConfig'
+import openExternalUrl from '../utils/openExternalUrl'
+import Link from './Link'
+import { SnackbarType } from './SnackbarContainer'
+import Checkbox from './base/Checkbox'
+
+const FlexContainer = styled.Pressable`
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+  flex: 1;
+`
+
+const StyledLink = styled(Link)`
+  color: ${props => props.theme.colors.linkColor};
+  text-decoration: underline solid ${props => props.theme.colors.linkColor};
+  align-self: center;
+`
+
+const StyledLabel = styled.Text`
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+  padding: 4px;
+  cursor: pointer;
+`
+
+type PrivacyCheckboxProps = {
+  language: string
+  checked: boolean
+  setChecked: (checked: boolean) => void
+  showSnackbar: (snackbar: SnackbarType) => void
+}
+
+const PrivacyCheckbox = ({ language, checked, setChecked, showSnackbar }: PrivacyCheckboxProps): ReactElement => {
+  const { privacyUrls } = buildConfig()
+  const { t } = useTranslation('common')
+  const privacyUrl = privacyUrls[language] || privacyUrls.default
+  return (
+    <FlexContainer onPress={() => setChecked(!checked)}>
+      <StyledLabel>
+        <Trans i18nKey='common:privacyAgreement'>
+          This gets replaced
+          <StyledLink
+            url={privacyUrl}
+            onPress={() => openExternalUrl(link, showSnackbar)}
+            text={t('privacyAgreementLink')}>
+            by react-i18next
+          </StyledLink>
```

Comment:
:wrench: I experimented a little and apparently the problem why the trans component does not automatically use the original translation is that the `Link` component uses a `text` instead of a children `prop`. Accordingly, if you change `text` to `children` in `Link.tsx`, you can get rid of the `t('privacyAgreementLink')`:
```suggestion
          <StyledLink url={privacyUrl}>
            by react-i18next
          </StyledLink>
```

The onPress is not needed in any case, so please remove :)